client/src/components/leftSidebar/index.tsx

```diff
@@ -45,6 +49,15 @@ class LeftSideBar extends React.Component {
         {scatterplotXXaccessor && scatterplotYYaccessor ? (
           <DynamicScatterplot />
         ) : null}
+        {geneIsOpen ? (
+          <GeneInfo
+            geneSummary=""
+            geneName=""
+            gene=""
+            geneUrl=""
+            geneSynonyms={[]}
+          />
```

Comment:
Weird, is TS requiring us to call this component with the props that will be populated with redux?

Reply:
Yeah I think it looks so bad :(